src/mds/Server.cc

```diff
@@ -1085,9 +1085,13 @@ void Server::recover_filelocks(CInode *in, bufferlist locks, int64_t client)
  */
 void Server::recall_client_state(float ratio)
 {
-  int max_caps_per_client = (int)(g_conf->mds_cache_size * .8);
-  int min_caps_per_client = 100;
-
+  int max_caps_per_client = (int)(g_conf->mds_cache_size * g_conf->max_ratio_caps_per_client);
```

Comment:
@batrick 
updated it, but not sure, so get the content is appropriate in the new config